docs/netdata-agent/configuration/optimizing-metrics-database/change-metrics-storage.md

```diff
@@ -32,7 +32,7 @@ retention strategies as shown in the table below:
 
 You can change these limits in `netdata.conf`:
 
-```text
+```conf
```

Comment:
What is `conf`?

Reply:
I have been adding it since always, to indicate config files, but after a search I see it is not indicated as supported on gh for example